readme.md

```diff
@@ -126,11 +132,24 @@ const post = new Post();
 can(user, 'view', post);
 ```
 
-### cannot(instance, action, target)
+```js
+/*
```

Comment:
Examples in readme should only contain code. If there are comments, they should be in text before that example.